docs/schedule/examples/fares-v2.md

```diff
@@ -216,7 +216,167 @@ The single ride fare product shown below has both `cash` and `tap-to-ride` fare
+| weekends   | 0      | 0       | 0         | 0        | 0      | 1        | 1      | 20220101   | 20240101 |
+| anyday     | 1      | 1       | 1         | 1        | 1      | 1        | 1      | 20220101   | 20240101 |
+
+
+Specific time frames are created in `timeframes.txt`, including general service days (anytime, weekdays and weekends), and peak and off peak periods:
+
+* AM Peak: from 6 am to 10 am on weekdays
+* AM2PM Peak: from 6 AM to 9 AM and from 4 pm to 8 pm on weekdays
+* Not AM Peak: weekday time not included in AM Peak
+* Not AM2PM Peak: weekday time not included in AM2PM Peak
+
+[**timeframes.txt**](../../reference/#timeframestxt)
+
+| timeframe_group_id | start_time | end_time | service_id |
+|:------------------:|:----------:|:--------:|:----------:|
+|       anytime      |  00:00:00  | 24:00:00 |   anyday   |
+|      weekdays      |  00:00:00  | 24:00:00 |  weekdays  |
+|      weekends      |  00:00:00  | 24:00:00 |  weekends  |
+|     mnr_ampeak     |  06:00:00  | 10:00:00 |  weekdays  |
+|    mnr_notampeak   |  00:00:00  | 06:00:00 |  weekdays  |
+|    mnr_notampeak   |  10:00:00  | 24:00:00 |  weekdays  |
+|    mnr_am2pmpeak   |  06:00:00  | 09:00:00 |  weekdays  |
+|    mnr_am2pmpeak   |  16:00:00  | 20:00:00 |  weekdays  |
+|  mnr_notam2pmpeak  |  00:00:00  | 06:00:00 |  weekdays  |
+| mnr_notam2pmpeak   | 09:00:00   | 16:00:00 | weekdays   |
+| mnr_notam2pmpeak   | 20:00:00   | 24:00:00 | weekdays   |
+
+
+Each individual fare product is defined in `fare_products.txt`, in this example, only trips between zone 1 and 9 are listed. The full dataset would include a record for each  price defined by a time and zone combination. Additionally, the example only uses one fare media (i.e. paper ticket) but additional combinations could be created if prices would also vary depending on fare media.
+
+[**fare_products.txt**](../../reference/#fare_productstxt)
+
+| fare_product_id        | fare_product_name                  | fare_media_id | amount | currency |
+|------------------------|------------------------------------|---------------|--------|----------|
+| mnr_1:HUD-9_adult_peak | Outbound Adult Peak Zonal Fare     | paper         | 25.75  | USD      |
+| mnr_1:HUD-9_adult      | Outbound Adult Off Peak Zonal Fare | paper         | 19.25  | USD      |
+| mnr_HUD-9:1_adult_peak | Inbound Adult Peak Zonal Fare      | paper         | 25.75  | USD      |
+| mnr_HUD-9:1_adult      | Inbound Adult Off Peak Zonal Fare  | paper         | 19.25  | USD      |
+
+
+Finally, origin and destination areas and timeframes combinations are associated with the corresponding fare product in `fare_leg_rules.txt`. Here, trips starting or arriving in Zone 1 (i.e. `area_id=mnr_1`) during peak times are subject to a specific peak fare corresponding to the arrival and departure zones of the trip (i.e. `fare_product_id=mnr_1:HUD-9_adult_peak`).
```

Comment:
Recommendation (I think in English, "finally" might be more "I finally got to experience the REM" type of sentences, but I might be wrong as I am not a native English speaker!)

Replacing:

> Finally, origin and destination areas and timeframes combinations are associated with the corresponding fare product in `fare_leg_rules.txt`.


With:

> Lastly, the combinations of origin and destination areas, along with their respective timeframes are associated with ...

Reply:
Thanks for this! I really appreciate those checks as I'm also not a native English speaker :)